spec/rspec/matchers/built_in/contain_exactly_spec.rb

```diff
@@ -105,6 +105,10 @@ def array.send; :sent; end
     }.to fail_with(/expected collection contained/)
   end
 
+  it 'works with hashes' do
+    expect([{ :a => 1 }]).to_not contain_exactly({ :a => 1 }, { :a => 2 }) 
+  end
```

Comment:
What does this spec have to do with the rest of this PR?  Seems completely unrelated...